src/Commands/MakeCommand.php

```diff
@@ -19,6 +19,12 @@ public function handle()
             $this->option('stub')
         );
 
+        if($this->isReservedClassName($name = $this->parser->className())) {
```

Comment:
Changed from a double negative "!$this->ensureClassNameIsNotReserved()" to a single positive "$this->isReservedClassName"

Also, "ensureClassNameIsNotReserved" both performs an action (throwing the message) AND returns a boolean (weather or not the class is reserved). I generally like to have a method responsable for only one of those things.

Reply:
Great to know for future PRs. Thanks!